src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -51,6 +52,14 @@ public void sign(KeyPair signer) {
     mSignatures.add(signer.signDecorated(txHash));
   }
 
+  public static void setDefaultBaseFee(int baseFee) {
+    if (baseFee < MINIMUM_BASE_FEE) {
+      throw new IllegalArgumentException("Invalid baseFee value: " + baseFee);
```

Comment:
Can it say that it's invalid because it's smaller than minimum?

Reply:
yes, will update.